### PR TITLE
update test from yoga to zed in devstack

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

use latest devstack

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**TODOs**:

- [ ] squashed commits
- if necessary:
  - [ ] includes documentation
  - [ ] adds unit tests

/hold

### CODE REVIEW

File: hack/ci/create_devstack.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Copyright 2021 The Kubernetes Authors.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
+# may not use this file except in compliance with the License.
```
Comment (mnaser) [inline hack/ci/create_devstack.sh:6]: I think the CI got upset that you dropped this

**Review by mnaser — CHANGES_REQUESTED** [review]

File: hack/ci/create_devstack.sh (hunk 1 above)
Comment (jichenjc) [reply hack/ci/create_devstack.sh:6]: I submit this PR when due to not drink my tea that day~ :)

**Comment by tobiasgiese** [comment]
@jichenjc I think your tests will be green after https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1408 is merged

**Comment by jichenjc** [comment]
no, my problem is devstack encounter issue after switch to zed, I am comparing logs between CPO an CAPO 
as CPO already moved to zed and it works fine...

**Comment by jichenjc** [comment]
no idea why this happen, continue to check

Feb 18 10:14:10 capo-e2e-controller neutron-server[133808]: [00;36mINFO neutron.wsgi [[00;36m-[00;36m] [01;35m[00;36m10.0.3.15 "GET /networking/v2.0/extensions HTTP/1.1" status: 404  len: 195 time: 0.0027072[00m[00m

**Review by mnaser — CHANGES_REQUESTED** [review]
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/cloud-init/controller.yaml.tpl#L38-L40

is there any reason we're adding those explicit services like trunk etc?

**Comment by lentzi90** [comment]
I don't know all the details, but regarding trunk I think it is because we have at least one test that makes use of it.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/864f0d859237a9e6b7550c823f4ec041b0213ce3/test/e2e/suites/e2e/e2e_test.go#L277-L282

**Comment by dulek** [comment]
> https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/cloud-init/controller.yaml.tpl#L38-L40
> 
> is there any reason we're adding those explicit services like trunk etc?

This is how you enable neutron extensions in general.

**Comment by jichenjc** [comment]
>>>This is how you enable neutron extensions in general.

right, I have no idea why it's failing , no time to dig into detail though..

**Comment by dulek** [comment]
> > > > This is how you enable neutron extensions in general.
> 
> right, I have no idea why it's failing , no time to dig into detail though..

Sounds like a job for me, I'll check.

**Comment by mdbooth** [comment]
Closing in favour of #1598, which I may also close.

**Pull request closed without merging**